source/administration/changelog.md

```diff
@@ -2,6 +2,148 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released bi-monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
+## Release v3.8.0
+
+Release Date: April 17, 2017
+
+### Security Updates
+
+- Mattermost v3.8.0 contains multiple [security updates](http://about.mattermost.com/security-updates/). [Upgrading to Mattermost v3.8.0](http://docs.mattermost.com/administration/upgrade.html) is highly recommended.
+
+### Highlights
+
+#### Native iOS and Android Apps (Beta)
+- Second generation mobile apps, built using React Native, are available for beta testing on [iOS](https://mattermost-fastlane.herokuapp.com/) and [Android](https://play.google.com/apps/testing/com.mattermost.react.native).
```

Comment:
https://about.mattermost.com/a-native-mobile-experience-second-generation-mobile-apps-released-in-beta/